packages/react-canvas-core/src/CanvasEngine.ts

```diff
@@ -29,15 +29,18 @@ export class CanvasEngine<
 	protected eventBus: ActionEventBus;
 	protected stateMachine: StateMachine;
 
+	deregisterDefaultDeleteItemsAction: () => void;
+	deregisterDefaultZoomCanvasAction: () => void;
+
```

Comment:
would prefer this to be options in the constructor

Reply:
Just 2 parameters passed by the constructor or an interface with the values? (like NodeModelOptions)

Reply:
@dylanvorster I configured it to be options in the constructor